scripts/review_comments_retriever.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from bugbug.tools.code_review import PhabricatorReviewData, ReviewCommentsDB
+from bugbug.vectordb import QdrantVectorDB
+
+
+def main():
+    review_data = PhabricatorReviewData()
+    vector_db = QdrantVectorDB("diff_comments")
+    vector_db.setup()
+    comments_db = ReviewCommentsDB(vector_db)
+    # TODO: support resuming from where last run left off. We should run it from
+    # scratch only ones. Following runs should add only new comments.
```

Comment:
only "once"?

Reply:
Fixed in 1f3de3395c9ff9abec0cfb493d902fcc3e4b22d0